dev-vcs/git-pw/git-pw-2.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+if [[ "${PV}" == 99999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/getpatchwork/git-pw.git"
+else
+	SRC_URI="https://github.com/getpatchwork/git-pw/archive/${PV}.tar.gz -> $P.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="A tool for integrating Git with Patchwork"
+HOMEPAGE="https://github.com/getpatchwork/git-pw"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+DEPEND="
```

Comment:
These are probably all only RDEPEND, usually Python stuff is filled with runtime-only dependencies